lib/models/utils/unfix-type.js

```diff
@@ -1,4 +1,7 @@
 module.exports = function unfixType(t) {
```

Comment:
```suggestion
module.exports = function unfixType(t = '') {
```

@esbanarango would adding a default value of `''` work here?